.circleci/config.yml

```diff
@@ -0,0 +1,19 @@
+version: 2
+jobs:
+  build:
+    docker:
+      - image: circleci/node:6.11.1
+
+    steps:
+      - checkout
+      - restore_cache:
+          key: dependency-cache-{{ checksum "package.json" }}
```

Comment:
We might also want to add a second layer of cache restoration here.  If we change this to:

```yml
keys:
  - dependency-cache-{{ checksum "yarn.lock" }}
  - dependency-cache
```

Circle will try to find the specific cache key with the checksum.  If it can't find it, it will look for the latest cache key that starts with `dependency-cache` and so we can at least get a partial cache which should speed up builds for cache misses.